keps/sig-apps/2214-indexed-job/README.md

```diff
@@ -148,19 +211,36 @@ because work lists can be implemented in a startup script using the completion
 index as building block.
 * The semantics of an indexed Job are similar to a StatefulSet, in the sense
 that Pods have an associated index.
-However, the APIs have a major difference: a StatefulSet doesn't have completion
-semantics, as opposed to Jobs.
+However, the APIs have major differences:
+  - a StatefulSet doesn't have completion semantics, as opposed to Jobs.
+  - a StatefulSet creates pods serially, whereas Job creates all Pods in
+    parallel.
+  - a StatefulSet gives Pods stable hostnames, a Job doesn't.
 
 [indexed Job]: https://github.com/kubernetes/community/blob/b21d1b27c8c748bf81283c2d89cde2becb5f2709/contributors/design-proposals/apps/indexed-job.md
 
 ### Risks and Mitigations
 
-Jobs have a known issue in which more than one Pod can be started even if
-parallelism and completion are set to 1 ([reference]). In the case of indexed
-Jobs, this translates to more than one Pod having the same index.
+- More than one pod created per index.
+
+  Jobs have a known issue in which more than one Pod can be started even if
+  parallelism and completion are set to 1 ([reference]). In the case of indexed
+  Jobs, this translates to more than one Pod having the same index.
+  
+  Just like for existing Job patterns, workloads have to handle duplicates at the
+  application level.
 
-Just like for existing Job patterns, workloads have to handle duplicates at the
-application level.
+- Scalability and latency of DNS programming.
```

Comment:
Are we doing to create (headless) services for jobs or not?
If not - I don't really understand this whole point...

Reply:
We leave it to the user. But this is going to be the suggested pattern in the tutorial.